.circleci/config.yml

```diff
@@ -80,7 +83,7 @@ jobs:
           name: Install python
           command: pyenv install << parameters.python-version >> -s
 
-      - run: &pyenv-set-system-python
+      - run:
```

Comment:
Any reason for removing this anchor but not the other unused ones (e.g., `pyenv-install-python`, `restore-cache-pyenv`, and some others)?

Reply:
I was touching this line and not the others :smile: I can remove them from the others as well.